general/document_splitter.ipynb

```diff
@@ -0,0 +1,283 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "W-dlkukAsGuQ"
+   },
+   "source": [
+    "# Overview\n",
+    "\n",
+    "## What is Document AI?\n",
+    "The [Document AI](https://cloud.google.com/document-ai/docs) API is a document understanding solution that takes unstructured data, such as documents, emails, and so on, and makes the data easier to understand, analyze, and consume. The API provides structure through content classification, entity extraction, advanced searching, and more.\n",
+    "\n",
+    "In this tutorial, you focus on using the Document AI API with Python. The tutorial demonstrates how to use Document Splitter to parse a simple PDF document with multiple scanned files to separate documents on page logical boundaries.\n",
+    "\n",
+    "## What you'll learn\n",
+    "- How to enable the Document AI API\n",
+    "- How to authenticate API requests\n",
+    "- How to install the client library for Python\n",
+    "- How to parse data from a multipage document and detect page logical boundaries\n",
+    "\n",
+    "## What you'll need\n",
+    "- A Google Cloud Project\n",
+    "- A Browser, such as Chrome or Firefox\n",
+    "- Knowledge of Python 3\n",
+    "- An instance of AI Notebook"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "5k6pF4jXuru9"
+   },
+   "source": [
+    "# Setup and Requirements\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "oLYSsjaJzbqX"
+   },
+   "source": [
+    "## Install the client library\n",
+    "Install the client library:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "tvNkaxakzkAf"
+   },
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "pip3 install --upgrade google-cloud-documentai\n",
+    "pip3 install --upgrade google-cloud-storage"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You should see something like this:\n",
+    "\n",
+    "```\n",
+    "...\n",
+    "Installing collected packages: google-cloud-documentai\n",
+    "Successfully installed google-cloud-documentai-0.3.0\n",
+    ".\n",
+    ".\n",
+    "Installing collected packages: google-cloud-storage\n",
+    "Successfully installed google-cloud-storage-1.35.0\n",
+    "```\n",
+    "\n",
+    "Now, you're ready to use the Document AI API!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "LPZhvzwesGvB"
+   },
+   "source": [
+    "# Document Splitter (Private Beta)"
```

Comment:
Remove "Private Beta" or this entire section of text. Replace with a simple title "Set your Processor Variables" as done in the other two notebooks.

Reply:
done